pyam/utils.py

```diff
@@ -423,9 +425,11 @@ def conv_strs(strs_to_convert, conv_codes, name):
 def datetime_match(data, dts):
     """Matching of datetimes in time columns for data filtering"""
     dts = dts if islistable(dts) else [dts]
-    if any([not isinstance(i, datetime.datetime) for i in dts]):
+    if any([not (
+            isinstance(i, datetime.datetime) or isinstance(i, np.datetime64)
```

Comment:
similar comment here on grouping together, but maybe [numpy has a check for both anyway](https://stackoverflow.com/questions/23063362/consistent-way-to-check-if-an-np-array-is-datetime-like/39020498)?

Reply:
I don't think so, datetime is not clearly related to datetime64